[2016-05-04 07:29:34] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'use' (T_USE), expecting ',' or ';'' in C:\xampp\htdocs\erp\api\app\Http\Controllers\OfficeController.php:10
Stack trace:
#0 {main}  
[2016-05-04 08:05:52] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'App\Http\Controllers\DB' not found' in C:\xampp\htdocs\erp\api\app\Http\Controllers\OfficeController.php:28
Stack trace:
#0 {main}  
[2016-05-04 08:32:42] local.ERROR: exception 'PDOException' with message 'SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'Hyderabad' for key 'office_updates_name_unique'' in C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php:358
Stack trace:
#0 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php(358): PDOStatement->execute(Array)
#1 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php(612): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'insert into `of...', Array)
#2 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php(576): Illuminate\Database\Connection->runQueryCallback('insert into `of...', Array, Object(Closure))
#3 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php(359): Illuminate\Database\Connection->run('insert into `of...', Array, Object(Closure))
#4 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php(316): Illuminate\Database\Connection->statement('insert into `of...', Array)
#5 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Query\Processors\Processor.php(30): Illuminate\Database\Connection->insert('insert into `of...', Array)
#6 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1718): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `of...', Array, 'id')
#7 [internal function]: Illuminate\Database\Query\Builder->insertGetId(Array, 'id')
#8 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(933): call_user_func_array(Array, Array)
#9 C:\xampp\htdocs\erp\api\vendor\compiled.php(9807): Illuminate\Database\Eloquent\Builder->__call('insertGetId', Array)
#10 C:\xampp\htdocs\erp\api\vendor\compiled.php(9807): Illuminate\Database\Eloquent\Builder->insertGetId(Array, 'id')
#11 C:\xampp\htdocs\erp\api\vendor\compiled.php(9797): Illuminate\Database\Eloquent\Model->insertAndSetId(Object(Illuminate\Database\Eloquent\Builder), Array)
#12 C:\xampp\htdocs\erp\api\vendor\compiled.php(9754): Illuminate\Database\Eloquent\Model->performInsert(Object(Illuminate\Database\Eloquent\Builder), Array)
#13 C:\xampp\htdocs\erp\api\app\Http\Controllers\OfficeController.php(35): Illuminate\Database\Eloquent\Model->save()
#14 [internal function]: App\Http\Controllers\OfficeController->activate_office()
#15 C:\xampp\htdocs\erp\api\vendor\compiled.php(8287): call_user_func_array(Array, Array)
#16 C:\xampp\htdocs\erp\api\vendor\compiled.php(8354): Illuminate\Routing\Controller->callAction('activate_office', Array)
#17 C:\xampp\htdocs\erp\api\vendor\compiled.php(8333): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\OfficeController), Object(Illuminate\Routing\Route), 'activate_office')
#18 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\erp\api\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#21 C:\xampp\htdocs\erp\api\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#22 C:\xampp\htdocs\erp\api\vendor\compiled.php(8334): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#23 C:\xampp\htdocs\erp\api\vendor\compiled.php(8320): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\OfficeController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'activate_office')
#24 C:\xampp\htdocs\erp\api\vendor\compiled.php(7317): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'activate_office')
#25 C:\xampp\htdocs\erp\api\vendor\compiled.php(7288): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\erp\api\vendor\compiled.php(6954): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#27 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\erp\api\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\erp\api\app\Http\Middleware\Auth.php(39): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): App\Http\Middleware\Auth->handle(Object(Illuminate\Http\Request), Object(Closure))
#31 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 C:\xampp\htdocs\erp\api\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\erp\api\vendor\compiled.php(6955): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#34 C:\xampp\htdocs\erp\api\vendor\compiled.php(6944): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#35 C:\xampp\htdocs\erp\api\vendor\compiled.php(6929): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#36 C:\xampp\htdocs\erp\api\vendor\compiled.php(1935): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#37 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#38 C:\xampp\htdocs\erp\api\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\erp\api\app\Http\Middleware\VerifyCsrfToken.php(18): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#40 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#41 C:\xampp\htdocs\erp\api\vendor\compiled.php(12083): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#42 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#43 C:\xampp\htdocs\erp\api\vendor\compiled.php(10785): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 C:\xampp\htdocs\erp\api\vendor\compiled.php(11789): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#46 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#47 C:\xampp\htdocs\erp\api\vendor\compiled.php(11738): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#48 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#49 C:\xampp\htdocs\erp\api\vendor\compiled.php(2478): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#51 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#52 C:\xampp\htdocs\erp\api\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#53 C:\xampp\htdocs\erp\api\vendor\compiled.php(1891): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#54 C:\xampp\htdocs\erp\api\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#55 C:\xampp\htdocs\erp\api\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#56 {main}

Next exception 'Illuminate\Database\QueryException' with message 'SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'Hyderabad' for key 'office_updates_name_unique' (SQL: insert into `office_updates` (`office`, `created_by`, `name`, `address`, `active`, `updated_at`, `created_at`) values (1, 1, Hyderabad, CM Camp Office, 0, 2016-05-04 08:32:42, 2016-05-04 08:32:42))' in C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php:620
Stack trace:
#0 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php(576): Illuminate\Database\Connection->runQueryCallback('insert into `of...', Array, Object(Closure))
#1 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php(359): Illuminate\Database\Connection->run('insert into `of...', Array, Object(Closure))
#2 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php(316): Illuminate\Database\Connection->statement('insert into `of...', Array)
#3 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Query\Processors\Processor.php(30): Illuminate\Database\Connection->insert('insert into `of...', Array)
#4 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1718): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `of...', Array, 'id')
#5 [internal function]: Illuminate\Database\Query\Builder->insertGetId(Array, 'id')
#6 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(933): call_user_func_array(Array, Array)
#7 C:\xampp\htdocs\erp\api\vendor\compiled.php(9807): Illuminate\Database\Eloquent\Builder->__call('insertGetId', Array)
#8 C:\xampp\htdocs\erp\api\vendor\compiled.php(9807): Illuminate\Database\Eloquent\Builder->insertGetId(Array, 'id')
#9 C:\xampp\htdocs\erp\api\vendor\compiled.php(9797): Illuminate\Database\Eloquent\Model->insertAndSetId(Object(Illuminate\Database\Eloquent\Builder), Array)
#10 C:\xampp\htdocs\erp\api\vendor\compiled.php(9754): Illuminate\Database\Eloquent\Model->performInsert(Object(Illuminate\Database\Eloquent\Builder), Array)
#11 C:\xampp\htdocs\erp\api\app\Http\Controllers\OfficeController.php(35): Illuminate\Database\Eloquent\Model->save()
#12 [internal function]: App\Http\Controllers\OfficeController->activate_office()
#13 C:\xampp\htdocs\erp\api\vendor\compiled.php(8287): call_user_func_array(Array, Array)
#14 C:\xampp\htdocs\erp\api\vendor\compiled.php(8354): Illuminate\Routing\Controller->callAction('activate_office', Array)
#15 C:\xampp\htdocs\erp\api\vendor\compiled.php(8333): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\OfficeController), Object(Illuminate\Routing\Route), 'activate_office')
#16 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\erp\api\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\erp\api\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 C:\xampp\htdocs\erp\api\vendor\compiled.php(8334): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#21 C:\xampp\htdocs\erp\api\vendor\compiled.php(8320): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\OfficeController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'activate_office')
#22 C:\xampp\htdocs\erp\api\vendor\compiled.php(7317): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'activate_office')
#23 C:\xampp\htdocs\erp\api\vendor\compiled.php(7288): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\erp\api\vendor\compiled.php(6954): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\erp\api\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\erp\api\app\Http\Middleware\Auth.php(39): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): App\Http\Middleware\Auth->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 C:\xampp\htdocs\erp\api\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\erp\api\vendor\compiled.php(6955): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#32 C:\xampp\htdocs\erp\api\vendor\compiled.php(6944): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\erp\api\vendor\compiled.php(6929): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#34 C:\xampp\htdocs\erp\api\vendor\compiled.php(1935): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#35 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#36 C:\xampp\htdocs\erp\api\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#37 C:\xampp\htdocs\erp\api\app\Http\Middleware\VerifyCsrfToken.php(18): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#39 C:\xampp\htdocs\erp\api\vendor\compiled.php(12083): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#40 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#41 C:\xampp\htdocs\erp\api\vendor\compiled.php(10785): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#42 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#43 C:\xampp\htdocs\erp\api\vendor\compiled.php(11789): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 C:\xampp\htdocs\erp\api\vendor\compiled.php(11738): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#46 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#47 C:\xampp\htdocs\erp\api\vendor\compiled.php(2478): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#48 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#49 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 C:\xampp\htdocs\erp\api\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#51 C:\xampp\htdocs\erp\api\vendor\compiled.php(1891): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#52 C:\xampp\htdocs\erp\api\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#53 C:\xampp\htdocs\erp\api\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#54 {main}  
[2016-05-04 12:28:01] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use App\User as User because the name is already in use' in C:\xampp\htdocs\erp\api\app\Http\Controllers\UsersController.php:9
Stack trace:
#0 {main}  
[2016-05-04 12:54:31] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '$response' (T_VARIABLE)' in C:\xampp\htdocs\erp\api\app\Http\Controllers\UsersController.php:26
Stack trace:
#0 {main}  
[2016-05-04 12:55:11] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function App\Http\Controllers\sha256()' in C:\xampp\htdocs\erp\api\app\Http\Controllers\UsersController.php:46
Stack trace:
#0 {main}  
[2016-05-04 12:56:05] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ')'' in C:\xampp\htdocs\erp\api\app\Http\Controllers\UsersController.php:46
Stack trace:
#0 {main}  
